src/components/compress/custom-els/MultiPanel/index.ts

```diff
@@ -1,19 +1,75 @@
-import './styles.css';
+import * as style from './styles.css';
+import { transitionHeight } from '../../../../lib/util';
 
-function getClosestHeading(el: Element) {
-  const closestEl = el.closest('multi-panel > *');
-  if (closestEl && closestEl.classList.contains('panel-heading')) {
-    return closestEl;
+interface CloseAllOptions {
+  exceptFirst?: boolean;
+}
+
+const openOneOnlyAttr = 'open-one-only';
+
+function getClosestHeading(el: Element): HTMLElement | undefined {
+  // Look for the child of multi-panel, but stop at interactive elements like links & buttons
+  const closestEl = el.closest('multi-panel > *, a, button');
+  if (closestEl && closestEl.classList.contains(style.panelHeading)) {
+    return closestEl as HTMLElement;
   }
   return undefined;
 }
 
+async function close(heading: HTMLElement) {
+  const content = heading.nextElementSibling as HTMLElement;
+
+  // if there is no content, nothing to expand
+  if (!content) return;
+
+  const from = content.getBoundingClientRect().height;
+
+  heading.removeAttribute('content-expanded');
+  content.setAttribute('aria-expanded', 'false');
```

Comment:
I've added animation to the component, but also changed how the attributes work. `aria-expanded` stays on the content as before, but `content-expanded` is added to the heading. This makes it easier to style the heading based on the expanded state of the content.